tutorial.md

```diff
@@ -0,0 +1,191 @@
+- IBM cloud CLI (IBM cloud command line interface)
+- kubectl (kubernetes command line interface)
+- Kubernetes Cluster
+
+---
+
+### If you already have the above pre-requisites, you can jump to [Step-by-Step](#stepbystep)
+
+---
+
+### 0. Make sure you created the accounts highlighted in [Pre-Requisites](#prereq), they are DockerHub and IBM Cloud Account, and install helm.
+
+### 1. Install IBM cloud CLI interface ibmcloud 
+```
+curl -fsSL https://clis.cloud.ibm.com/install/osx | sh
+```
+Note: This instructions assumes user has Mac OSX. To install IBM Cloud CLI in another platform please visit [platforms](https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli) and check your OS equivalent command line option to install ibmcloud CLI
+
+### 2. Install kubernetes service plugin (might need to run with sudo) kubectl
+```
+ibmcloud plugin install kubernetes-service
+```
+
+### 3. <a name="step3"></a>In a folder of your choosing Clone this project hello cloud
+```
+git clone https://github.com/cloudreadyapp/hello_cloud_demo.git
+cd hello_cloud_demo
+```
+
+### 4. Login into your IBM account
+```
+ibmcloud login
+```
+Note: If there’s an error message: “ou are using a federated user ID, please use one time passcode ( ibmcloud login --sso )” use
+```
+ibmcloud login --sso
+```
+
+### 5.0 Create a kubernetes cluster (You can also do it via the dashboard UI at https://cloud.ibm.com). NOTE: You can only have 1 Lite Kubernetes cluster running at a time. If you already have one already, use the existing one or delete it.
+```
+ibmcloud ks cluster create classic --name hello-cloud-app-cluster
+```
+Note: If creating cluster for the first time, it will take a while, ~20 minutes. Cluster is ready once it's in normal state.
+### 5.1 <a name="checkStatus"></a>To check the status of the cluster, type
+```
+ibmcloud ks clusters
+```
+
+### 6. Create a Cloudant Database. You can do so by following this link [here](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-using-the-ibm-cloudant-dashboard)
+
+Before continuing with the tutorial, make sure you have the appropriate `server.env` file under `database/src/main/liberty/config/server.env` with the appropriate database constants that you created in step 6. It should look something like
+```
+cloudantapikey=<cloundant-api-key>
+cloudantendpoint=https://<cloundant-end-point>-bluemix.cloudantnosqldb.appdomain.cloud
+cloudantdbname=cloud-app
+```
+
+---
+
+# <a name="stepbystep"></a>Step-by-Step
+
+### Before proceding, make sure your cluster is in normal state as highlighted in [step 6](#checkStatus) above
+
+---
+
+### 7.0 Make sure you're in the folder from the cloned repository from [step 3](#step3)
+
+### 7.1 Install and setup necessary maven packages
+```
+mvn -f database install
+mvn -f database liberty:package
+docker pull open-liberty
+```
+
+### 8. Create container image with tag name (database is the microservice we’re creating)
+```
+docker build -t hello-cloud-app database
```

Comment:
should this be hello-cloud-app-image?

Reply:
The suffix `-image` is not really used in this context for naming container images

Reply:
hello-cloud-app-image is used in Step 9, it seems we should be using one or the other of: hello-cloud-app or hello-cloud-app-image